src/cf-buttons/usage.md

```diff
@@ -348,77 +348,152 @@ For accessibility reasons, use the semantic `<button>` instead of a link when po
 
 #### Button with icon on the left
 
-<button class="a-btn a-btn_icon__left cf-icon cf-icon-delete">
-    Close
+<button class="cf-icon
+               cf-icon__before
+               cf-icon-delete
+               a-btn
+               a-btn__icon-on-left">
```

Comment:
No real opinion but looking at it, it makes sense since we're defining buttons here and the icon is a modifier.